### PR TITLE
[PERFSCALE-2349]: Loaded Migration - OpenShift SDN to OVNK

### PR DESCRIPTION
**Pull request by krishvoor**
This PR allows users to run the "load" on the OCP via OpenShift SDN
and eventually, move OVNK, allowing users to understand what potentially issues might can get uncovered.

### CODE REVIEW

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-sdn-to-ovn-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.13-ocp-qe-perfscale-aws-sdn-to-ovn-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.12-ocp-qe-perfscale-aws-sdn-to-ovn-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-data-path-9nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-sdn-to-ovn-24nodes-periodic

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.12-ocp-qe-perfscale-aws-sdn-to-ovn-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.13-ocp-qe-perfscale-aws-sdn-to-ovn-24nodes-periodic

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 1]
```diff
@@ -85,6 +85,306 @@ periodics:
...
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-qe
+    ci-operator.openshift.io/variant: aws-4.12
+    ci.openshift.io/generator: prowgen
+    job-release: "4.12"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.12-ocp-qe-perfscale-aws-sdn-to-ovn-120nodes-periodic
```
[the remark this answers is not in the document]
Comment (krishvoor) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:105]: @svetsa-rh : sure will add them

**Comment by jtaleric** [comment]
Any reason to have an explicit timeout here 
```
+ oc wait mcp --all --for=condition=UPDATING=True --timeout=300s
timed out waiting for the condition on machineconfigpools/master
timed out waiting for the condition on machineconfigpools/worker
```

We could just use the CI system timeout instead of the `300`?

**Comment by jtaleric** [comment]
https://github.com/openshift/release/blob/master/ci-operator/step-registry/ovn/sdn-migration/ovn-sdn-migration-commands.sh#L33 

We might need to get with the OVN team to make these timeouts vars instead of statically set.

**Comment by krishvoor** [comment]
> Any reason to have an explicit timeout here
> 
> ```
> + oc wait mcp --all --for=condition=UPDATING=True --timeout=300s
> timed out waiting for the condition on machineconfigpools/master
> timed out waiting for the condition on machineconfigpools/worker
> ```
> 
> We could just use the CI system timeout instead of the `300`?

These were the defaults, will update the logic, incase timeout is not set, will default to `300`

**Comment by vishnuchalla** [comment]
> > Any reason to have an explicit timeout here
> > ```
> > + oc wait mcp --all --for=condition=UPDATING=True --timeout=300s
> > timed out waiting for the condition on machineconfigpools/master
> > timed out waiting for the condition on machineconfigpools/worker
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > We could just use the CI system timeout instead of the `300`?
> 
> These were the defaults, will update the logic, incase timeout is not set, will default to `300`

Fix PR: https://github.com/openshift/release/pull/42583

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-x86-perfscale-rosa-sdn-to-ovn-24nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.12-ocp-qe-perfscale-aws-sdn-to-ovn-24nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-perfscale-aws-sdn-to-ovn-24nodes-periodic

**Comment by jtaleric** [comment]
It doesn't appear you are taking advantage of the timeout param that @vishnuchalla added? 

https://github.com/openshift/release/blob/master/ci-operator/step-registry/ovn/sdn-migration/ovn-sdn-migration-ref.yaml#L9

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-perfscale-aws-sdn-to-ovn-24nodes-periodic

**Comment by krishvoor** [comment]
Recent failure from this [URL](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/42306/rehearse-42306-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-perfscale-aws-sdn-to-ovn-24nodes-periodic/1714077870759874560) is not related to the PR in itself, however, due to a test failure, will retrigger the job.

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-perfscale-aws-sdn-to-ovn-24nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-perfscale-aws-sdn-to-ovn-24nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-sdn-to-ovn-24nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-x86-perfscale-rosa-sdn-to-ovn-24nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-perfscale-aws-sdn-to-ovn-24nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.12-ocp-qe-perfscale-aws-sdn-to-ovn-24nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-x86-perfscale-rosa-sdn-to-ovn-24nodes-periodic

**Comment by krishvoor** [comment]
The recent [job-logs](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/42306/rehearse-42306-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-x86-perfscale-rosa-sdn-to-ovn-24nodes-periodic/1715006414377717760/artifacts/perfscale-rosa-sdn-to-ovn-24nodes-periodic/rosa-cluster-provision/build-log.txt) needs to be a OpenshiftSDN type and needs to be explicitly set.
```
  "network": {
    "host_prefix": 23,
    "machine_cidr": "10.0.0.0/16",
    "pod_cidr": "10.128.0.0/14",
    "service_cidr": "172.30.0.0/16",
    "type": "OVNKubernetes"
  }
```

**Comment by krishvoor** [comment]
This is blocked by https://github.com/openshift/release/pull/44633

**Comment by krishvoor** [comment]
Switching to the approach of validating it on Self-Managed AWS.

**Pull request closed without merging**